Profile Folder/chrome/locale/uk/dtd/gsettings.dtd

```diff
@@ -66,7 +66,7 @@
 <!ENTITY gsettings.toolbar						"Панель Інструментів">
 <!ENTITY gsettings.needsRestart					"Потребує перезапуск">
 <!ENTITY gsettings.tabManager					"Менеджер вкладок">
-<!ENTITY gsettings.tabManagerDescription		'[!!! Éñáβℓèƨ ƭλè "ářřôω βúƭƭôñ" ωïƭλ ƭλè ℓïƨƭ ôƒ ôƥèñ ƭáβƨ ïñ ƭλè ƭáββář !!!]'>
+<!ENTITY gsettings.tabManagerDescription		'Вмикає "кнопку стрілочку" зі списком відкритих вкладок у панелі вкладок'>

```

Comment:
Would it be possible to find a way to not use "Enable" in `gsettings.tabManagerDescription` and `gsettings.enableForceColorizeAddressBar`? The toggle already implies that it's for "turning on".